packages/pwa-kit-react-sdk/src/ssr/universal/components/_error/index.test.js

```diff
@@ -15,16 +15,9 @@ describe('Error Page', () => {
     const message = 'Error message'
 
     test('Renders correctly', () => {
-        const wrapper = mount(<Error message={message} stack={stack} status={status} />)
-
-        expect(wrapper.props().message).toBe(message)
-        expect(wrapper.props().stack).toBe(stack)
-        expect(wrapper.props().status).toBe(status)
-    })
-
-    test('Ensure that status type is a number', () => {
```

Comment:
We intentionally removed this test since React Testing library do not focus on the prop type, rather than focus on what components are rendering, and we want to see on the DOM.